azurerm/data_source_loadbalancer_backend_address_pool.go

```diff
@@ -33,6 +33,7 @@ func dataSourceArmLoadBalancerBackendAddressPool() *schema.Resource {
 			"backend_ip_configurations": {
 				Type:     schema.TypeList,
 				Optional: true,
```

Comment:
Any attribute that isn't used to get the resource should remove the `Required/Optional: true` tag in favor of `Computed: true`

Reply:
Thanks, will sort that out.